### ISSUE TITLE
Twitter cards and opengraph metadata should be enabled in more user-facing views

### ISSUE THREAD

**Issue by mjgiarlo** [projectblacklight/spotlight#1504]
Spotlight already supports Twitter cards and opengraph metadata (for rich social media snippets) in exhibit homepages; it would be awfully nice to have the same for other views such as the browse view, about pages, and feature pages. Blocked by #1503

Originally surfaced as part of sul-dlss/exhibits#218
